http/file_server_test.ts

```diff
@@ -284,8 +317,8 @@ Deno.test("checkURIEncodedPathTraversal", async function () {
       "http://localhost:4507/%2F..%2F..%2F..%2F..%2F..%2F..%2F..%2F..",
     );
 
-    assertEquals(res.status, 404);
-    const _ = await res.text();
+    assertEquals(res.status, 200);
+    assertStringIncludes(await res.text(), "README.md");
```

Comment:
Correct me if I'm wrong, but I think this test was not correct... or simply I don't understand what it tests.

Reply:
Looks like this was the test for a security fix for path traversal vulnerability in very old version of file_server https://github.com/denoland/deno/pull/8134/files

and that attack doesn't seems possible now (even with this fix) because we process the path with `new URL().pathname` and `posix.normalize(path)`

I think this change is ok